GooglePlayInstant/Editor/PlayInstantAssetBundleBrowserClient.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    public static class AssetBundleBrowserClient
+    {
+        private const string AssetBundleBrowserName = "com.unity.assetbundlebrowser";
+        private const string AssetBundleBrowserMenuItem = "Window/AssetBundle Browser";
+        private static bool? _assetBundleBrowserIsPresent;
+        private static string _assetBundleBrowserVersion;
+
+        /// <summary>
+        /// Whether or not Asset Bundle Browser is present
+        /// </summary>
+        public static bool AssetBundleBrowserIsPresent
+        {
+            get { return BundleBrowserIsPresent(); }
+        }
+
+        /// <summary>
+        /// The detected version of Asset Bundle Browser.
+        /// </summary>
+        public static string AssetBundleBrowserVersion
```

Comment:
Delete this method and make the one it calls public? It doesn't seem to do anything beyond the one it calls

Reply:
Resolved